wbtc-ethereum-form.yaml

```diff
@@ -0,0 +1,12 @@
+form:
```

Comment:
Did you mean to add these config files in the monorepo root? Seems like the wrong place for them

Reply:
ups no they leaked here from a warp deployment, removing them